suffuse/node_fuse.go

```diff
@@ -3,22 +3,92 @@ package suffuse
 import (
   "github.com/suffuse/go-suffuse/suffuse/xattr"
   "os"
+  "time"
   "golang.org/x/net/context"
   "bazil.org/fuse/fs"
   f "bazil.org/fuse"
 )
 
+// Access checks whether the calling context has permission for
```

Comment:
> There are many things that could 'in principle' be handled by software, for example indentation in languages that use { and } for code blocks or whitespace at the end of the line.

Yes, 100% those should be handled by software. Not "in principle" either, in fact. In my case whitespace at the end of the line IS handled by software, I'll never commit it because I never generate it. I'd prefer we all did that.